### ISSUE TITLE
Fix layout of widgets in the dialogue “New Computed Column”

### ISSUE THREAD

**Issue by Markus-Elfring** [eclipse/birt#905]
I am trying [the software “birt-report-designer-all-in-one-4.9.0-20220315-linux.gtk.x86_64.zip”](https://download.eclipse.org/birt/downloads/drops/latest/birt-report-designer-all-in-one-4.9.0-20220315-linux.gtk.x86_64.zip "Update candidate") out.

I suggest to improve the layout for buttons in [the dialogue “New Computed Column”](https://eclipse.github.io/birt-website/docs/t_data-sets#data-set-editor-pages "Documentation").

Questionable screenshot:
![questionable_layout-New_Computed_Column-dialogue-20220329](https://user-images.githubusercontent.com/102525885/160606584-79e29033-6de9-42cf-88d3-73ffaa31e082.png)

**Comment by hvbtup** [comment]
This seems to be a Linux-specific issue. On Windows, the dialog layout is OK:
![grafik](https://user-images.githubusercontent.com/7644776/160607970-6d782a8e-eb1f-412b-90e7-9317afd7a88a.png)

**Comment by claesrosell** [comment]
I am running Fedora 35 and I cannot either reproduce this issue. @Markus-Elfring can you reproduce this problem consistantly?
Do you have a report design were this always happens? If so, can you provide us with it?
![image](https://user-images.githubusercontent.com/8941815/160713240-b7ab13ff-bfd6-4edb-9d55-3f07abc34824.png)

**Comment by Markus-Elfring** [comment]
>can you reproduce this problem consistantly?

Yes. ‒ I am observing the improvable display for two buttons in the mentioned dialogue (within a KDE session for example) on an openSUSE Tumbleweed system.
The reproduction is possible here after the selection of a data set for another report designer test so that the dialogue “New Computed Column” can be opened accordingly.

**Comment by wimjongman** [comment]
I know there are sometimes issues with this type of dialog when font scaling is used in the OS. This notably happens when OS fonts are scaled e.g. to 125%

**Comment by Markus-Elfring** [comment]
:thought_balloon: I imagine that other software developers became more familiar with involved implementation details (than me so far).
I am unsure how the chances will evolve for corresponding contributions.

**Comment by wimjongman** [comment]
> I know there are sometimes issues with this type of dialog when font scaling is used in the OS. This notably happens when OS fonts are scaled e.g. to 125%

Did you check if you have scaling set?

**Comment by wimjongman** [comment]
Closing as stale. Reopen if you want to work on it.
